Editors/IncludeEditor.cpp

```diff
@@ -0,0 +1,14 @@
+#include "IncludeEditor.h"
+#include "ui_IncludeEditor.h"
+
+IncludeEditor::IncludeEditor(QWidget *parent) :
+  QWidget(parent),
+  ui(new Ui::IncludeEditor)

```

Comment:
Why isn't this some kind of smart pointer?

Reply:
Qt autogenerated that from the class creator and Qt objects use the Qt ownership semantics.
https://doc.qt.io/qt-5/objecttrees.html

If you're proposing we make it a smart pointer, we can, but I'm not so sure it's a good idea we go against the Qt conventions. We might shoot ourselves in the foot that way.